ui/ui-app/src/main/resources/static/locales/en.json

```diff
@@ -814,12 +814,26 @@
     "Monitors All Service Health":"Monitors All Service Health",
     "Total":"Total",
 
+    "service-component-detail.time-locale": "{{date|date:'dd/MM/yyyy @ H:mm'}}",
```

Comment:
This should match the original format: `"service-component-detail.time-locale": "{{date|date:\"MM/dd/yyyy 'at' h:mm\"}}",`